### PR TITLE
Cleanup the tests a bit

### PR DESCRIPTION
**Pull request by spaantje**
Hi @christophrumpel, Now that your application is growing i took the liberty to structure the testcases a bit. Most of these changes are highly opinionated. A summary of the changes: 

- Moved testcase to a structure to match the `app/` structure as close as possible.
- Change the `Test` class names so you can use the "Go to test" (**⇧⌘T**) hotkey in PHPStorm to navigate. 
- Used the factory states when available. 
- Use the Route::group & Route::view instead of a closure. 
- Removed some fakes which where not used. 
- Extracted the youtube responses to a trait and only use the trait in the tests where they where used.   

Like i said this may be _highly opinionated_ so if you don't like this feel free to decline.

### CODE REVIEW

**Comment by christophrumpel** [comment]
Thanks @spaantje, I will check asap.

File: tests/Unit/FeedTest.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Tests\Feature;
+namespace Tests\Unit;
```
Comment (christophrumpel) [inline tests/Unit/FeedTest.php:3]: What makes this a `Unit` test for you?

**Review by christophrumpel — COMMENTED** [review]
Hey,

thanks for the work. Looks good and more consistent now. The test structure with `http` and `controllers` I'm not used to, but I will give it a try.  I think I only left one question for you to check.

**Comment by spaantje** [comment]
@christophrumpel In response to "The test structure with http and controllers I'm not used to"
It's the structure that JMac uses in the TestGenerator of Laravel Shift. And that way it's easier to see which files do not have a testcase yet.

File: tests/Unit/FeedTest.php (hunk 1 above)
Comment (spaantje) [reply tests/Unit/FeedTest.php:3]: I don't really know; just a preference/feeling thing.. (Also see comment below)

Comment (christophrumpel) [reply tests/Unit/FeedTest.php:3]: kk, then please change this back to a Feature test. For me, everything that uses HTTP is a feature test, only when I test a class / method in isolation it is a unit test for me.

**Comment by christophrumpel** [comment]
Thanks a lot @spaantje !

**Pull request merged**